src/Orleans/SystemTargetInterfaces/IMembershipTable.cs

```diff
@@ -19,8 +20,7 @@ public interface IMembershipTable
         /// </summary>
         /// <param name="globalConfiguration">the give global configuration</param>
         /// <param name="tryInitTableVersion">whether an attempt will be made to init the underlying table</param>
-        /// <param name="logger">the logger used by the membership table</param>
-        Task InitializeMembershipTable(GlobalConfiguration globalConfiguration, bool tryInitTableVersion, Logger logger);
+        Task InitializeMembershipTable(GlobalConfiguration globalConfiguration, bool tryInitTableVersion);
```

Comment:
removed Logger from IMembershipTable.Init it now gets logger from constructor injection